### PR TITLE
clbs-v1.25.56

### PR DESCRIPTION
**Pull request by cody-somerville**
Add documentation for cipher profile feature and release notes for v1.25.56.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK PREVIEW

* [`api-docs/cloud-load-balancers-v2`](https://staging.developer.rackspace.com/build-7c7028a458/docs/cloud-load-balancers/v2/)
* [`api-docs/cloud-load-balancers-v1`](https://staging.developer.rackspace.com/build-7c7028a458/docs/cloud-load-balancers/v1/)

**Comment by cody-somerville** [comment]
This PR supersedes #179 and #178 by combining them into commit 8bcddd2 in this PR.

**Comment by cyrichardson** [comment]
Will edit today, @cody-somerville. Thanks!

File: api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+SSL_RSA_WITH_AES_256_GCM_SHA384           *             *
+SSL_RSA_WITH_AES_256_CBC_SHA256           *             *
+SSL_RSA_WITH_AES_256_CBC_SHA              *             *
+SSL_RSA_WITH_AES_128_GCM_SHA256           *             *
+SSL_RSA_WITH_AES_128_CBC_SHA256           *             *
+SSL_RSA_WITH_AES_128_CBC_SHA              *             *
+SSL_RSA_WITH_3DES_EDE_CBC_SHA             *
+====================================== ======= ======================
+
+Table. Ciphers statuses
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst]: Cipher statuses (delete Table.)

Add an intro sentence similar to "The following table shows the cipher statuses:

File: api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
++--------------------------+-------------------------+-------------------------------+
+|Name                      |Type                     |Description                    |
++==========================+=========================+===============================+
+|ciphers                   |Object                   |A list of named ciphers        |
+|                          |                         |associated to the load         |
+|                          |                         |balancer.                      |
++--------------------------+-------------------------+-------------------------------+
+
+**Example List Ciphers: JSON response**
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst]: List ciphers

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 3]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+|cipherProfile             |String                   |Specifies a cipher       |
+|                          |                         |profile to be used. The  |
+|                          |                         |default value is         |
+|                          |                         |``default``.             |
++--------------------------+-------------------------+-------------------------+
+
+.. note::
+
+   CipherProfile's are defined by the service and are immutable.
+   The service currently offers two profiles. The 'default' profile, which
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: The ``default`` profile is set on any load balancer enabling SSL termination by default.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 4]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+   is set on any load balancer enabling SSLTermination by default. The second
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: The second profile that you can optionally select is ``CLBCipherPolicy2017-08``.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 5]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+   profile that can optionally be selected is ``CLBCipherPolicy2017-08``.
+   :ref:`Ciphers <ciphers>` section for more details about available profiles.
+
+.. note::
+
+   In order to remove a ``cipherProfile`` set ``cipherProfile``
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: In order to remove a ``cipherProfile``, set

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 6]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+   to ``default`` when updating
+   :ref:`SSLTermation <put-update-ssl-termination-configuration>`.
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: SSL termination

File: api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v1.0/{account}/loadbalancers/{loadbalancerid}/ssltermination/ciphers
+
+Lists load balancer ciphers that are configured for the load balancer.
+
+The response object shows a list of ciphers defined for the load balancer.
+
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst]: Insert an intro sentence for the table, similar to "The following table shows the possible response codes for this operation:"

**Comment by MadhaviLReddy** [comment]
A couple of trivial changes that might be considerate, observed these in the HULK PREVIEW:

- In api-reference/ssl-termination/,  _**The second profile that can optionally be selected is CLBCipherPolicy2017-08. Ciphers section for more details about available profiles**._ 
We can add "See:" similar to the next paragraph.

- In api-reference/ciphers/, _**'*' denotes the cipher belonging to the related profile.**_
But the cipher profile table is using a bullet, we can try to use same marker in both places, either * or a bullet.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 8]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
 +--------------------------+-------------------------+-------------------------+
+|cipherProfile             |String                   |Specifies a cipher       |
+|                          |                         |profile to be used. The  |
+|                          |                         |default value is         |
+|                          |                         |``default``.             |
++--------------------------+-------------------------+-------------------------+
+
+.. note::
+
+   CipherProfile's are defined by the service and are immutable.
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst (hunk 3 above)
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst (hunk 4 above)
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 9]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+   The service currently offers two profiles. The 'default' profile, which
+   is set on any load balancer enabling SSLTermination by default. The second
+   profile that can optionally be selected is ``CLBCipherPolicy2017-08``.
+   :ref:`Ciphers <ciphers>` section for more details about available profiles.
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst (hunk 5 above)
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst (hunk 6 above)
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 10]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+
+.. note::
+
+   In order to remove a ``cipherProfile`` set ``cipherProfile``
+   to ``default`` when updating
+   :ref:`SSLTermation <put-update-ssl-termination-configuration>`.
+   All SSLTermination objects are built using the default profile.
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst [hunk 11]
```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
...
+   See: :ref:`Ciphers <ciphers>` section for more details about available
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst]: Removed this entire section.

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst [hunk 12]
```diff
@@ -0,0 +1,34 @@
+.. version-v1.25.56-release-notes:
+
+v1.25.56, March 13th 2018
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst]: March 13th 2018 is the date the actual release was deployed. We are behind on getting documentation update out.

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+-----------
+
+- Feature: Cipher Profile support
+
+  - Loadbalancers with SSL termination can now optionally assign a pre-defined cipher profile to change the set of
+    ciphers enabled for negotiation between clients and the loadbalancer.
+  - By default, loadbalancers are assigned the ``default`` cipher profile which is managed by Rackspace and updated
+    from time to time to disable ciphers that have become insecure. For this reason, use of the ``default`` cipher
+    profile is recommended.
+  - If you have a loadbalancer with a cipher profile assigned that uses insecure or weak ciphers, it is highly
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst]: Removed this section. Moved content to ciphers page with suggestion accepted.

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    recommended that you assign an updated cipher profile or re-assign the ``default`` profile.
```
[the remark this answers is not in the document]
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst]: Removed this section. Moved content to ciphers page with suggestion accepted.

**Comment by rackernexus** [comment]
HULK PREVIEW

* [`api-docs/cloud-load-balancers-v1`](https://staging.developer.rackspace.com/build-7c7028a458/docs/cloud-load-balancers/v1/)
* [`api-docs/cloud-load-balancers-v2`](https://staging.developer.rackspace.com/build-7c7028a458/docs/cloud-load-balancers/v2/)

**Comment by cody-somerville** [comment]
@cyrichardson The HULK PREVIEW is incorrect. Since I amended my previous commit it seems to have simply reused the old build. Any way to force it to rebuild?

File: api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst (hunk 1 above)
Comment (cody-somerville) [reply api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst]: @the2hill Hey, can you comment here?

Comment (the2hill) [reply api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst]: It's an example of the available ciphers. Similar to how aws does it and is what was suggested.

**Comment by rackernexus** [comment]
HULK PREVIEW

* [`api-docs/cloud-load-balancers-v2`](https://staging.developer.rackspace.com/build-bb8ebc3fc7/docs/cloud-load-balancers/v2/)
* [`api-docs/cloud-load-balancers-v1`](https://staging.developer.rackspace.com/build-bb8ebc3fc7/docs/cloud-load-balancers/v1/)

**Comment by cyrichardson** [comment]
So sorry, @cody-somerville! I was on PTO last week and just caught up with your question here. I made a small change to your branch that forced a new build. When you have time, can you please check to see if your changes are reflected in it? Thanks!

**Comment by cyrichardson** [comment]
Done, @cody-somerville. Thanks!

File: api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+By default, load balancers are assigned the ``default`` cipher profile which is
+managed by Rackspace and updated from time to time to disable ciphers that have
+become insecure. For this reason, use of the ``default`` cipher profile is
+recommended.
+
+.. warning::
+   Only the ``default`` profile is updated to remove insecure ciphers.
+   If you have a load balancer with a cipher profile assigned that uses insecure
+   or weak ciphers, we highly recommended that you assign an updated cipher
+   profile or re-assign the ``default`` profile.
```
Comment (filler) [inline api-docs/cloud-load-balancers-v1/api-reference/ciphers.rst]: Question on this warning block.  If the `default` profile is updated over time to remove insecure ciphers, why does the table list `default` as having the 🔥 `3DES` 🔥 cipher in it while `CLBCipherPolicy2017-08` does not?

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+    from time to time to disable ciphers that have become insecure. For this reason, use of the ``default`` cipher
+    profile is recommended.
+  - Other profiles are added and managed by Rackspace. As security concerns change, new cipher profiles may be added.
+    Included with this release is the availability of the ``CLBCipherPolicy2017-08`` cipher profile which currently
+    matches the ``default`` profile with the exception of disabling the ``SSL_RSA_WITH_3DES_EDE_CBC_SHA`` cipher.
+    Rackspace security experts have determined the aforementioned cipher is still secure for use with Cloud Load
+    Balancers due to maximum lifetime of requests but due to being downgraded from a "high" to "medium" strength cipher
+    will be removed from the ``default`` profile in the near future. However, if you would like to disable it now you
+    can leverage this new feature and the ``CLBCipherPolicy2017-08`` profile to have it disabled on your load balancers
+    immediately.
```
Comment (filler) [inline api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-25-56-20180313.rst]: I _think_ this clarifies the state behind the comment on https://github.com/rackerlabs/docs-cloud-load-balancers/pull/188/files#r181415298 ...

**Pull request merged**